generators/app/templates/npmrc

```diff
@@ -1 +1 @@
-registry=http://registry.npmjs.org/
+package-lock=false
```

Comment:
This file can be deleted, since `package-lock.json` is ignored

Reply:
See https://github.com/kentcdodds/generator-kcd-oss/pull/45#discussion_r524247939